engine/config/1.0/schema.json

```diff
@@ -229,76 +230,14 @@
                   "client_secret": {
                     "type": "string",
                     "description": "The OAuth client secret of the auth provider."
-                  },
-                  "oauth2": {
```

Comment:
Bad copy-paste on my part -- this shouldn't be part of the well-known provider config path. (it is ignored by the Engine)